be/src/runtime/runtime_state.cpp

```diff
@@ -421,4 +420,26 @@ int64_t RuntimeState::get_load_mem_limit() const {
     return 0;
 }
 
+const vectorized::GlobalDictMaps& RuntimeState::get_global_dict_map() const {
+    return _global_dicts;
+}
+
+Status RuntimeState::init_global_dict(const GlobalDictLists& global_dict_list) {
+    for (const auto& global_dict : global_dict_list) {
+        DCHECK_EQ(global_dict.ids.size(), global_dict.strings.size());
+        vectorized::GlobalDictMap dict_map;
+        vectorized::RGlobalDictMap rdict_map;
+        int dict_sz = global_dict.ids.size();
+        for (int i = 0; i < dict_sz; ++i) {
+            auto str = _obj_pool->add(new std::string(global_dict.strings[i]));
```

Comment:
Could we allocate a large buffer to put all strings into it?  Looks like there are a lot of objects in object pool.